icicle/appUtils/ntt/kernel_ntt.cu

```diff
@@ -1025,9 +1025,27 @@ namespace ntt {
     bool columns_batch,
     bool is_inverse,
     bool fast_tw,
-    Ordering ordering,
+    ntt::Ordering ordering,
     curve_config::scalar_t* arbitrary_coset,
     int coset_gen_index,
     cudaStream_t cuda_stream);
 
-} // namespace ntt
+  // #if defined(ECNTT_DEFINED)
```

Comment:
remove commented code

Reply:
left commented as requested by @LeonHibnik

Reply:
consider writing a comment explaining why it is commented out